crates/kind-openai/src/endpoints/chat.rs

```diff
@@ -199,6 +199,15 @@ impl<T> ChatCompletionChoice<T> {
 struct ChatCompletionResponseMessage<T> {
     content: T,
     refusal: Option<String>,
+    usage: Usage,
```

Comment:
Add a `usage` method on `ChatCompletionChoice` that gives us `&Usage`

Reply:
I didn't realize that `usage` is under a different key so this wasn't the right place for it.